assets/minecraft/models/item/mob/hisomukuro/hisomukuro_swim0.json

```diff
@@ -0,0 +1,581 @@
+{
```

Comment:
put all these frames in a folder either named `excluded` inside of the mob. Anything inside that folder will not be added to the resourcepack on the server. Also unzip any zip files like `hisomukuro` and do the same for them